app/assets/javascripts/discourse/tests/integration/components/d-stat-tiles-test.gjs

```diff
@@ -0,0 +1,66 @@
+import { render, triggerEvent } from "@ember/test-helpers";
+import { module, test } from "qunit";
+import DStatTiles from "discourse/components/d-stat-tiles";
+import { setupRenderingTest } from "discourse/tests/helpers/component-test";
+import { i18n } from "discourse-i18n";
+
+module("Integration | Component | DStatTiles", function (hooks) {
+  setupRenderingTest(hooks);
+
+  test("formats the @value in a readable way with the raw number as the title attr", async function (assert) {
+    await render(<template>
+      <DStatTiles as |tiles|><tiles.Tile
+          @value="12555999"
+          @label={{i18n "bootstrap_mode"}}
+        /></DStatTiles>
```

Comment:
would this formatting in tests be more readable? wdyt? (prettier might put the Tile on a single line then, could be even better, or worse 😄 dunno)

```suggestion
      <DStatTiles as |tiles|>
        <tiles.Tile
          @value="12555999"
          @label={{i18n "bootstrap_mode"}}
        />
      </DStatTiles>
```